docs/diagrams/RemarkLogicClassDiagram.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2020-10-27T10:19:40.397Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.111 Safari/537.36" etag="Vf1kbqBrDfxT7fcRB6GN" version="13.8.3" type="device"><diagram id="pWqK2U7bCta3QDFsQahJ" name="Page-1">7VpLk5s4EP41vmzVuHgJ8HFemaQ2WzuVOWw2NxmEIQHECnnG3l+/EhJgQNhkAuwkhS9GLaklur+vJbVYmbfJ4YHALPwD+yheGZp/WJl3K8PQNdNlf1xyFBLgAiHYkciXjWrBU/QvKntK6T7yUd5oSDGOaZQ1hR5OU+TRhgwSgl+azQIcN0fN4A51BE8ejLvSvyKfhkLqGk4tf4+iXViOrNsbUZPAsrF8kzyEPn45EZn3K/OWYEzFU3K4RTE3XmkX0e9dT201MYJSOqTDgxHoye/vvn75/OUj1R+u80/+hyvdkJOjx/KNkc8MIIuY0BDvcArj+1p6Q/A+9RFXq7FS3eYjxhkT6kz4FVF6lN6Ee4qZKKRJLGvZjMnx82nhb65sDcri3UEqF6WjLIm58gn22kCKcrwnHjr34hJLkOwQPdMOVJ5iEEc4QWw+rB9BMaTRc3MeUGJtV7Wr3cEepEe+xztC7zOM93KkjreavngJI4qeMli8+gtjZNPuQRTHtzjGpOhr+gC5vsXkOSX4GzqpcY2taduVxZ8Roehw3uZdG8kOpikJICOAbsvyS80nvSRJeMIloE1kVkthVjum3ECYvdOpfe1/9risuMoLPF+zBgbIDnUle9rx/0cGCW4UqYzNTegTtR3XMZPSpn8IYkPAbdGAu5MzRwxaVMM42qXs2WODIOapG+6XiMWqa1mRRL5fMDTDUUoLs4GbFbhruT7FKep4XQqbeBJNysDnjAQICzQB4XQBsVHgwZgKD2BimkHkBp6KZrbnom0wjlXtt8YyeyKWfUIJJN8Wkp2Hg9OCg4JjujYnyZxpSRYEgeEpSebbWxuMtJYBt2VWsOmaFYAZWaabP9MGDh0iyrtdaWvGbikQXR1HFuuuvHDa8xGRiFmNs3Ls7aA7cDvYA495toPutPuWG4yXsHoxABjry7sXHcwZWfX+/WyewXQILnRNhQt2pqYwSvMTUAiFCygaoDCs9qrgdCAxLyIGLLYnXqoyE9xCPszDylzDHcR0ZFxzctjxRNBa5F4M8c/VFoF8zdfhIjlkcfVBHGXv5SRSTL1Q6SXFaq8Vv5G81zmfWoqtUtd7+lTeM1TH/oXPMwZ57Y3xuaTvwucB3rM2b43PZi+ftzX3vm8jp1sqgmvrtd67g6vE24XwLcjo7WOd2z3WWbMSvn9LNzJkflvw8ooQ0062bbpnAFWubTq8qFKYk+BliS+vyXi3NxSK+DIvXvqTswte/n+8APDW8DIkexviZLtnI1/K3M66GzS1TdOUDhhkSn0zlSlVWbwxE+Guh9SJ8K0LLDCSWduJcFOVBJszD25seiNaRlA76gw+Rxf5UmEzragLYBLFR1HLusEkKypNk9+jhyh+RjzedGoqJYUzr3LhTa4kxYVLFddc0P8zjY/q5Gzz3C4jN1EE18bLL+FVgNBxWklcSwFfWxEVrKngaw741AOl/rU4fZe2qs72etON5TWLvJAp71hW5y5YxHCjXKbIxeLiZYppqN10Ic1SygZfusgRHjkmT1aGVp6gujItVYj3lL1q/3YU2cBtLjFaS5EwREdRAZTqtX8AO79cdohgyjyL+ZAbTR07RggDtuWunWYccBVxoIoVs+y0zP5k0c+5jlUq4m4p4jgLuIpS3OzSXsyWVe+HPhBqX10aCrSrMqOvWPVYsf6yVQS6+vtg8/4/</diagram></mxfile>
```

Comment:
Hmm I've looked into using plantUML for the class diagram. However, the way plantUML generates its class diagram may not be accepted by our module due to its unconventional layout. Not really sure if I remembered correctly from one of our tutorial sessh as the following layout by plantUML looked familiar to a rejected version.

![image](https://user-images.githubusercontent.com/48002577/97406551-5b8ea180-1934-11eb-9b86-572d62a25a50.png)

Reply:
We can use the UML diagram similar to the current diagrams in our DG. Cos the methods are not really required.
You can refer to `LogicClassDiagram.puml` and `ModelClassDiagram.puml` for the labels and the multiplicity.

Because your current diagram looks very similar to what we have in our DG, just that the font size and colors are not standardised